tests/functional/adapter/incremental/test_incremental_strategies.py

```diff
@@ -65,15 +67,16 @@ def seeds(self):
     def test__bigquery_assert_incremental_configurations_apply_the_right_strategy(self, project):
         run_dbt(["seed"])
         results = run_dbt()
-        assert len(results) == 11
+        assert len(results) == 12
 
         results = run_dbt()
-        assert len(results) == 11
+        assert len(results) == 12
         incremental_strategies = [
             ("incremental_merge_range", "merge_expected"),
             ("incremental_merge_time", "merge_expected"),
             ("incremental_overwrite_time", "incremental_overwrite_time_expected"),
             ("incremental_overwrite_date", "incremental_overwrite_date_expected"),
+            ("incremental_overwrite_date_with_comment_at_end", "incremental_overwrite_date_expected"),
```

Comment:
The linter (`black`) found this:
```
-            ("incremental_overwrite_date_with_comment_at_end", "incremental_overwrite_date_expected"),
+            (
+                "incremental_overwrite_date_with_comment_at_end",
+                "incremental_overwrite_date_expected",
+            ),
```
Could you make this update and push again?

Reply:
@mikealfare 
Sorry, I installed pre-commit and fixed it.